### ISSUE TITLE
Ideas for the project

### ISSUE THREAD

**Issue by MrJmpl3** [AmreshSinha/Spotify-Cards-API#54]
Hi! I have some ideas about the project:

- Text color (same a background color but with the text)
- Sort the files of the project
- Use prettier and huskyjs (to format the code and get a standard codelines)
- Use Vuejs and Vite or Webpack (That depends is you want)

If you want, I can do that things

**Comment by AmreshSinha** [comment]
- ~Text color~. This API is for generating cards just like spotify sharing cards in its app. You can also see it here: https://promocards.byspotify.com/?type=song&id=4EpdhdsbhHQ3CXXuqqwHyQ
- Sort. Can you elaborate on this more?
- Use prettier and huskyjs. I also feel that adding husky for pre-commit hooks will be cool and time saver for everyone.
- Use Vuejs and Vite or Webpack. Are you referring to the documentation part?

**Comment by MrJmpl3** [comment]
Yes, to the documentation part.

Another idea:
- Move the documentation to another branch or repo

**Comment by AmreshSinha** [comment]
Yeah, it would be great, but I feel that this project does not need extensive documentation. We can use React or Vue for making simple documentation instead of using powerful options like docusaurus.

**Comment by AmreshSinha** [comment]
I have created a separate issue for adding husky and prettier. Add a comment there so that I can assign you.
